modules/ROOT/pages/lb-cert-validation.adoc

```diff
@@ -53,13 +55,165 @@ If your load balancer already exists, you can use the https://anypoint.mulesoft.
+          "description": "Default ciphers (new version)",
+          "defaultSuite": true
+      },
+      {
+          "id": "5b4a36e1e65c892316abd4d2",
+          "name": "OldDefault",
+          "ciphers": "ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-SHA:ECDHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES128-SHA256:DHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA:AES256-GCM-SHA384:AES128-GCM-SHA256:AES256-SHA256:AES128-SHA256:AES256-SHA:AES128-SHA:HIGH:!aNULL:!eNULL:!EXPORT:!DES:!MD5:!PSK:!RC4",
+          "description": "Default ciphers (old version)",
+          "defaultSuite": false
+      },
+      {
+          "id": "5b4bfedee4b0f45ab1de6979",
+          "name": "EcdheEcdsaRsa",
+          "ciphers": "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384",
+          "description": "ECDHE ECDSA RSA ciphers",
+          "defaultSuite": false
+      },
+      {
+          "id": "5ba28106e4b00522d78f40b6",
+          "name": "RestrictCiphers",
+          "ciphers": "AES256-GCM-SHA384:AES128-GCM-SHA256:AES256-SHA256:AES128-SHA256",
+          "description": "Restricted Ciphers",
+          "defaultSuite": false
+      }
+  ],
+  "total": 4
+}
+----
+
+=== Display the Default Cipher Suite for a DLB 
+
+To display the default cipher suite for a DLB using Cloudhub API, run this GET call:
+
+[source,linenums]
+----
+curl https://anypoint.mulesoft.com/cloudhub/api/organizations/myOrgID/vpcs/myVpcID/loadbalancers/myDlbID -H 'Authorization: Bearer myAccessToken' | jq
+----
+
+In the command: 
+
+* _myOrgID_ is the Organization ID.
+* _myVpcID_ is the ID for the VPC.
+* _myDlbID_ is the ID for the DLB.
+* _myAccessToken_ is the access token you generated.
+
+The output includes `defaultSuite`, for example:
+
+[source,linenums]
+----
+"defaultCipherSuite": "ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-SHA:ECDHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES128-SHA256:DHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA:AES256-GCM-SHA384:AES128-GCM-SHA256:AES256-SHA256:AES128-SHA256:AES256-SHA:AES128-SHA:HIGH:!aNULL:!eNULL:!EXPORT:!DES:!MD5:!PSK:!RC4"
+----
+
+=== Rotate the Cipher by Replacing the Default Cipher Suite
+
+To rotate the cipher, replace the default cipher suite for a DLB using Cloudhub API, run this GET call:
+
+[source,json,linenums]
+----
+curl -X PATCH https://anypoint.mulesoft.com/cloudhub/api/organizations/myOrgID/vpcs/myVpcID/loadbalancers/myDlbID  -H 'Authorization: Bearer myAccessToken' -H 'Content-Type: application/json' -d '[{"op":"replace", "path":"/defaultCipherSuiteName", "value":"NewDefault"}]' | jq
+----
+
+In the command: 
+
+* _myOrgID_ is the Organization ID.
```

Comment:
I should have asked this earlier: do we really need to capitalize Organization in this repeated list item?